src/azure-cli/azure/cli/command_modules/storage/operations/account.py

```diff
@@ -111,6 +111,14 @@ def list_storage_accounts(cmd, resource_group_name=None):
     return list(accounts)
 
 
+def show_storage_accounts(cmd, client, resource_group_name, account_name, expand=None, include_restore_status=None):
+    if include_restore_status:
+        StorageAccountExpand = cmd.get_models('StorageAccountExpand')
+        expand = StorageAccountExpand()
```

Comment:
If `expand` and `include_restore_status` pass in at the same time, is it in line with expectations that `expand` will be overwritten?

Reply:
I will remove this part and the change in another PR. Thanks for reminder.